ext/fileinfo/libmagic.patch

```diff
@@ -1170,7 +1170,7 @@ diff -u libmagic.orig/compress.c libmagic/compress.c
  # endif /* HAVE_SIG_T */
  #endif 
 -#if !defined(__MINGW32__) && !defined(WIN32)
```

Comment:
`__MINGW32__` appears to target a compiler infrastructure, including standard library and headers which may or may not (probably not) contain `<sys/ioctl.h>`, where-as my Windows 7 system with MSYS2 installed reports the presense of `<sys/ioctl.h>`. Is it possible that you might have confused a compiler-related macro with an OS-related macro?